plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialDirectional.java

```diff
@@ -14,6 +14,8 @@
 import org.bukkit.block.BlockFace;
 import org.bukkit.block.data.*;
 import org.bukkit.block.data.type.PointedDripstone;
+import org.bukkit.block.data.type.Jigsaw;
+import org.bukkit.block.data.type.Jigsaw.Orientation;

```

Comment:
... wot?

Reply:
presumably leftover from when jigsaw was planned to be in this property